lib/global.dart

```diff
@@ -1,4 +1,6 @@
 // ignore: constant_identifier_names
 const BASE_URL = "https://dev-v2.tabletop-cloud.com";
+
+
 // const BASE_URL = "https://prod-v2.tabletop-cloud.com";
 // const BASE_URL = "https://stage-v2.tabletop-cloud.com";
```

Comment:
@GauravCalidig I think we can remove this `BASE_URL` and the it's use in `ChatUtils`'s constructor